### DISCUSSION TITLE
Import CSV

### DISCUSSION THREAD

**Discussion by jheredianet** [teslamate-org/teslamate#2155] [Q&A] [answered]
I'm importing several CSV files. When I put all the files it imports by takes its time, but when I import let's say 3 files, after the import and restart I doesn't import any more files. Is it something programmed by default? I mean, if you finish importing your files, you cannot import any more files?

Thanks for any help.

**Comment by adriankumpf** [comment, marked as answer]
Only files are imported, which are chronologically before the existing data. If a blue arrow is displayed during import, this means that the respective file was skipped (not imported).
